drivers/hmis/lib/form_data/default/fragments/r20_post_ext_information_date.json

```diff
@@ -0,0 +1,20 @@
+{
+  "type": "GROUP",
+  "link_id": "r20.postExitInformationDate",
+  "data_collected_about": "HOH_AND_ADULTS",
+  "text": "Post-Exit Details",
+  "item": [
+    {
+      "type": "DATE",
+      "link_id": "R20.1",
+      "required": false,
+      "warn_if_empty": true,
+      "text": "Aftercare Information Date",
+      "assessment_date": true,
+      "mapping": {
+        "record_type": "EXIT",
+        "field_name": "aftercareDate"
+      }
```

Comment:
[data dictionary says](https://files.hudexchange.info/resources/documents/HMIS-Data-Dictionary-2024.pdf)
> Information may be entered post exit for a period of up to 180 days after which
point no data should be entered. Any data entered prior to the date of exit or after
the 180th day will not be considered in reporting or exports.


Can we add a value bound constraint with `$exitDate` (min) `$exitDate + 180 days` (max)? There is a way to specify the number of offset days in the JSON, look for `offset`. The exitDate should probably already be present as a "local constant" when the form is rendered. let me know if you have any questions/issues!